pkgs/development/libraries/nfd/default.nix

```diff
@@ -0,0 +1,64 @@
+{ stdenv, fetchgit, doxygen, boost, pkgconfig, python, pythonPackages, ndn-cxx, graphviz, libpcap, openssl }:
+let
+  version = "3651fd";
+in
+stdenv.mkDerivation {
+  name = "nfd-0.2-${version}";
+  src = fetchgit {
+    url = "https://github.com/named-data/NFD.git";
+    rev = "3651fd75f79c0f8cb2bf67193d4ef581cebaee8c";
+    sha256 = "18lk6vpwpys8189jbjcrhhqhz6pmqxjimkfkx9fhszhgv8mcbgy5";
+  };
+  buildInputs = [ doxygen boost pkgconfig openssl python pythonPackages.sphinx ndn-cxx graphviz libpcap];
+  preConfigure = ''
+    patchShebangs ./waf
+    ./waf configure \
+      --boost-includes=${boost.dev}/include \
+      --boost-libs=${boost.lib}/lib \
+      --with-libpcap=${libpcap} \
+      --sysconfdir=/etc/ndn \
+      --prefix=$out
+  '';
+  buildPhase = ''
+    ./waf
+  '';
+  installPhase = ''
+    mkdir /etc/ndn
```

Comment:
nix packages cannot install things to /etc. Putting things in /etc is something that must be done in nixos modules.